src/style.css

```
@@ -1 +1,46 @@
 /* add styles here */
+
+body {
+ margin: 0;
+}
+
+.star__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.star__item {
+  width: 16px;
+  height: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.star__item:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .star__item:nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 .star__item:nth-child(-n + 2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--3 .star__item:nth-child(-n + 3) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 .star__item:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--5 .star__item:nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
+}
```

```suggestion
.stars--1 .star__item:nth-child(-n + 1),
.stars--2 .star__item:nth-child(-n + 2),
.stars--3 .star__item:nth-child(-n + 3),
.stars--4 .star__item:nth-child(-n + 4),
.stars--5 .star__item:nth-child(-n + 5) {
  background-image: url(/src/images/star-active.svg);
}
```

Combine selectors using comma